charmcraft/templates/init-flask-framework/charmcraft.yaml.j2

```diff
@@ -0,0 +1,31 @@
+# This file configures Charmcraft.
+# See https://juju.is/docs/sdk/charmcraft-config for guidance.
+
+name: {{ name }}
+
+bases:
+  - build-on:
+    - name: ubuntu
+      channel: "22.04"
+    run-on:
+    - name: ubuntu
+      channel: "22.04"
+
+# (Required)
+summary: A very short one-line summary of the flask application.
+
+# (Required)
+description: |
+  A comprehensive overview of your Flask application.
+
+extensions:
+  - flask-framework
+
+# Uncomment the integrations used by your application
+# requires:
+#   mysql:
+#     interface: mysql_client
+#     limit: 1
+#   postgresql:
+#     interface: postgresql_client
+#     limit: 1
```

Comment:
Should this include all of the integrations we support? E.g., ingress seems to be missing?

Reply:
Ingress and other relations are default and static for now, so user can't change them.

Reply:
Ah I see